### PR TITLE
Remove ELS for CentOS 7

### PR DESCRIPTION
**Pull request by timmullin**
Case RE-368:

Use the manual method to remove ELS. This involves deleting two files and un-installing the package.

Changelog: Removes ELS for CentOS 7.

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: lib/Elevate/Components/ELS.pm [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      /etc/yum.repos.d/centos7-els-rollout.repo
+    };
+
+    foreach my $file (@files_to_remove) {
+        if ( -e $file ) {
+            unlink $file or WARN("Could not remove file $file: $!");
+        }
+    }
+
+    $self->yum->remove(ELS_PACKAGE);
```
Comment (toddr) [inline lib/Elevate/Components/ELS.pm:42]: This is my best guess too but I'm uncertain if there's more than one package to remove :\

**Review by toddr — APPROVED** [review]

File: lib/Elevate/Components/ELS.pm (hunk 1 above)
Comment (timmullin) [reply lib/Elevate/Components/ELS.pm:42]: I got that from the manual steps outlined in the document referenced in the case.

Comment (cPholloway) [reply lib/Elevate/Components/ELS.pm:42]: The rest is replacing system packages.  My guess is that leapp handles replacing the rest with upgrades for A8.

**Review by cPholloway — APPROVED** [review]

File: lib/Elevate/Components/ELS.pm [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+use constant ELS_PACKAGE => 'els-define';
+
+sub pre_leapp ($self) {
+
+    return unless Elevate::OS::remove_els();
+
+    return unless Cpanel::Pkgr::is_installed(ELS_PACKAGE);
+
+    my @files_to_remove = qw{
+      /etc/yum/vars/elstoken
```
Comment (cPholloway) [inline lib/Elevate/Components/ELS.pm:31]: @timmullin - I just got an updated version of the install script for ELS and with IP based licensing, this file is not needed.  Can you remove it from the list of files to remove please?

Comment (timmullin) [reply lib/Elevate/Components/ELS.pm:31]: I can go ahead and remove that.

Comment (timmullin) [reply lib/Elevate/Components/ELS.pm:31]: Now that I think about it, what if els had been installed with the older version of the script and the file is present?  Plus, we check if the file exists before deleting it; so, the file's absence would not create an error condition for the removal of els if they had used the newer version of the script.  It doesn't seem that there is any harm in leaving it in.  And, there are probably plenty of installs out there that have used the older version of the script and ought to have that file removed.

**Review by cPholloway — APPROVED** [review]

File: lib/Elevate/Components/ELS.pm (hunk 1 above)
Comment (toddr) [reply lib/Elevate/Components/ELS.pm:41]: agreed!

**Review by annes449 — APPROVED** [review]
QA is good!

**Pull request merged**